pkg/kubelet/cm/dra/manager.go

```diff
@@ -595,6 +613,12 @@ func (m *Manager) unprepareResources(ctx context.Context, podUID types.UID, name
 
 	// Atomically perform some operations on the claimInfo cache.
 	err := m.cache.withLock(func() error {
+		// TODO(#132978): Re-evaluate this logic to support post-mortem health updates.
+		// As of the initial implementation, we immediately delete the claim info upon
+		// unprepare. This means a late-arriving health update for a terminated pod
+		// will be missed. A future enhancement could be to "tombstone" this entry for
+		// a grace period instead of deleting it.
```

Comment:
side note: I wonder if the solution here may be covering both device plugin and DRA